src/nodes.coffee

```diff
@@ -630,10 +630,10 @@ exports.AwaitReturn = class AwaitReturn extends Return
 # or vanilla.
 exports.Value = class Value extends Base
   constructor: (base, props, tag, isDefaultValue = no) ->
-    return base if not props and base instanceof Value
-
     super()
 
+    return base if not props and base instanceof Value
```

Comment:
Minor point, but this was deliberately before the `super` call as it's an early return so the call is redundant. Was something broken?